niflow_manager/cli/main.py

```diff
@@ -13,15 +18,34 @@ def main():
 
 
 @main.command()
-def install():
-    print("install command not yet implemented")
+@click.argument("workflow_path", type=click.Path(), default=".")
+def install(workflow_path):
+    print(f"installing {workflow_path}")
+    sp.check_call([sys.executable, "-m", "pip", "install", f"{workflow_path}"])
```

Comment:
The intended organization of this directory is that each command would go in a `<command>.py` file, and be integrated here here with:

```Python
from .test import test

main.command()(test)
```

See `init`.

Reply:
ok, got it